### PR TITLE
register an autoloader for proxies to avoid issues when unserialising

### PR DESCRIPTION
**Pull request by ikwattro**
See https://github.com/symfony/symfony/issues/3907 for references

Fixes https://github.com/graphaware/neo4j-php-ogm/issues/122

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@
 /phpspec.yml
 /phpunit.xml
 /vendor/
+.idea/
```
Comment (Nyholm) [inline .gitignore]: Should be in global .gitignore.

Comment (ikwattro) [reply .gitignore]: fixed

**Comment by ikwattro** [comment]
@Nyholm Tests passing, please make a 0.4.0 release when you got a spot. Thanks!

**Comment by Nyholm** [comment]
Awesome. Thank you

**Pull request merged**